roles/app-config/templates/curation_concerns.rb.j2

```diff
@@ -0,0 +1,65 @@
+CurationConcerns.configure do |config|
+  # Injected via `rails g curation_concerns:work Section`
+  config.register_curation_concern :section
+  # Injected via `rails g curation_concerns:work Monograph`
+  config.register_curation_concern :monograph
+  # Should schema.org microdata be displayed?
+  # config.display_microdata = true
+
+  # What default microdata type should be used if a more appropriate
+  # type can not be found in the locale file?
+  # config.microdata_default_type = 'http://schema.org/CreativeWork'
+
+  # How frequently should a file be audited.
+  # Note: In CurationConcerns you must trigger the FileSetAuditService manually.
+  # config.max_days_between_audits = 7
+
+  # Enable displaying usage statistics in the UI
+  # Requires a Google Analytics id and OAuth2 keyfile.  See README for more info
+  # config.analytics = false
+
+  # Specify a Google Analytics tracking ID to gather usage statistics
+  # config.google_analytics_id = 'UA-99999999-1'
+
+  # Specify a date you wish to start collecting Google Analytic statistics for.
+  # config.analytic_start_date = DateTime.new(2014,9,10)
+
+  # Where to store tempfiles, leave blank for the system temp directory (e.g. /tmp)
+  # config.temp_file_base = '/home/developer1'
+
+  # Location on local file system where derivatives will be stored.
+  # If you use a multi-server architecture, this MUST be a shared volume.
+  config.derivatives_path = '{{ derivatives_path }}'
+
+  # Location on local file system where uploaded files will be staged
+  # prior to being ingested into the repository or having derivatives generated.
+  # If you use a multi-server architecture, this MUST be a shared volume.
+  config.working_path = '{{ uploads_path }}'
```

Comment:
I worry a bit about templating this entire file rather than adding or overwriting the specific configuration settings that need to be changed from what's committed to github.  It seems like either a team might want to make changes to some of the settings that are currently commented out, or that curation concerns might continue evolving and the format or contents of this file might change in new versions of curation_concerns.